Multisig/Logic/Models/Transaction/MethodRegistry.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MethodRegistry.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 15.06.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+/// A namespace for value types representing methods of smart contracts
+enum MethodRegistry {
+
+    struct MethodSignature {
+        var name: String
+        var parameterTypes: [String]
+
+        init(_ name: String, _ params: String...) {
+            self.name = name
+            self.parameterTypes = params
+        }
+    }
+
+}
+
+func == (lhs: TransactionData, signature: MethodRegistry.MethodSignature) -> Bool {
```

Comment:
why "lhs & signature" but not "lhs & rhs" or "transaction & signature"?

Reply:
No specific reason, just a preference, I thought they are not the same types, so I'd name them different 😃